packages/suite-web-landing/pages/index.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import styled from 'styled-components';
+import Layout from '@suite-web-landing-components/Layout';
+import Download from '@suite-web-landing-components/Download';
+import Feature from '@suite-web-landing-components/Feature';
+import { resolveStaticPath } from '@suite-utils/nextjs';
+import { H1, P, variables, colors } from '@trezor/components';
+import { Fade } from 'react-awesome-reveal';
+
+const Wrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    font-size: ${variables.FONT_SIZE.NORMAL};
+`;
+
+const StyledHeroCta = styled.header`
+    text-align: center;
+    z-index: 1;
+`;
+
+const StyledCta = styled.div`
+    text-align: center;
+    justify-content: center;
+    margin: 168px 0 140px;
+`;
+
+const DownloadWrapper = styled.div`
+    display: flex;
+    justify-content: center;
+`;
+
+const FeaturesWrapper = styled.div`
+    margin: 80px 0 0 0;
+    & > section {
+        margin-bottom: 50px;
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+    }
+`;
+
+const StyledH1 = styled(H1)`
+    font-size: 28px;
+    line-height: 36px;
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+
+    @media only screen and (min-width: ${variables.SCREEN_SIZE.MD}) {
+        font-size: 44px;
+        line-height: 55px;
+        white-space: pre-wrap;
+    }
+`;
+
+const StyledHeadline = styled(H1)<{ size?: number }>`
+    font-size: 40px;
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    line-height: 1.3;
+    margin-bottom: 18px;
+    & > em {
+        font-style: normal;
+        color: ${colors.NEUE_TYPE_GREEN};
+    }
+
+    @media only screen and (min-width: ${variables.SCREEN_SIZE.MD}) {
+        font-size: ${props => (props.size !== undefined ? `${props.size}px` : '64px')};
+    }
+`;
+
+const StyledSubheadline = styled(P)`
+    font-size: 20px;
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    color: ${colors.BLACK50} !important;
+    margin-bottom: 65px;
+`;
+
+const StyledP = styled(P)`
+    && {
+        font-size: 20px;
+        line-height: 34px;
+        font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+        color: ${colors.BLACK50};
+    }
+`;
+
+const StyledSoon = styled.div`
+    font-size: 16px;
+    line-height: 24px;
+    text-transform: uppercase;
+    font-weight: ${variables.FONT_WEIGHT.BOLD};
+    color: ${colors.NEUE_TYPE_ORANGE};
+`;
+
+const features = [
+    {
+        id: 1,
+        headline: 'Desktop app. 2x safer.',
+        text:
+            'Security and privacy. Anti-Phishing, Bitcoin\nCore, Tor support, Discreet mode, Performance.',
+        backgroundPosition: 'bottom right',
```

Comment:
We don't have Bitcoin Core and Tor support atm, should be switch the soon boolean on? cc @matejzak

Reply:
Those are just copy placeholders and are there to illustrate the direction of that particular paragraph. Copy is to be updated after the freeze.